.changeset/real-stingrays-battle.md

```diff
@@ -0,0 +1,97 @@
+'@commercetools-uikit/primary-action-dropdown': patch
+'@commercetools-uikit/spacings-inset': patch
+'@commercetools-uikit/spacings-stack': patch
+'@commercetools-uikit/primary-button': patch
+'@commercetools-uikit/date-time-field': patch
+'@commercetools-uikit/date-time-input': patch
+'@commercetools-uikit/rich-text-input': patch
+'@commercetools-uikit/rich-text-utils': patch
+'@commercetools-uikit/password-field': patch
+'@commercetools-uikit/checkbox-input': patch
+'@commercetools-uikit/password-input': patch
+'@commercetools-uikit/flat-button': patch
+'@commercetools-uikit/icon-button': patch
+'@commercetools-uikit/link-button': patch
+'@commercetools-uikit/number-field': patch
+'@commercetools-uikit/select-field': patch
+'@commercetools-uikit/number-input': patch
+'@commercetools-uikit/select-input': patch
+'@commercetools-uikit/select-utils': patch
+'@commercetools-uikit/toggle-input': patch
+'@commercetools-uikit/collapsible-motion': patch
+'@commercetools-uikit/data-table-manager': patch
+'@commercetools-uikit/money-field': patch
+'@commercetools-uikit/radio-field': patch
+'@commercetools-uikit/input-utils': patch
+'@commercetools-uikit/money-input': patch
+'@commercetools-uikit/radio-input': patch
+'@commercetools-uikit/accessible-hidden': patch
+'@commercetools-uikit/collapsible-panel': patch
+'@commercetools-uikit/date-field': patch
+'@commercetools-uikit/text-field': patch
+'@commercetools-uikit/time-field': patch
+'@commercetools-uikit/date-input': patch
+'@commercetools-uikit/text-input': patch
+'@commercetools-uikit/time-input': patch
+'@commercetools-uikit/loading-spinner': patch
+'@commercetools-uikit/notifications': patch
+'@commercetools-uikit/view-switcher': patch
+'@commercetools-uikit/field-errors': patch
+'@commercetools-uikit/collapsible': patch
+'@commercetools-uikit/constraints': patch
+'@commercetools-uikit/field-label': patch
+'@commercetools-uikit/data-table': patch
+'@commercetools-uikit/pagination': patch
+'@commercetools-uikit/calendar-time-utils': patch
+'@commercetools-uikit/messages': patch
+'@commercetools-uikit/tooltip': patch
+'@commercetools-uikit/avatar': patch
+'@commercetools-uikit/icons': patch
+'@commercetools-uikit/label': patch
+'@commercetools-uikit/stamp': patch
+'@commercetools-uikit/card': patch
+'@commercetools-uikit/grid': patch
+'@commercetools-uikit/link': patch
+'@commercetools-uikit/text': patch
+'@commercetools-uikit/localized-utils': patch
+'@commercetools-local/generator-package-json': patch
+'@commercetools-uikit/calendar-utils': patch
+'@commercetools-uikit/tag': patch
+'@commercetools-local/generator-readme': patch
+'@commercetools-uikit/spacings': patch
+'@commercetools-uikit/buttons': patch
+'@commercetools-uikit/hooks': patch
+'@commercetools-uikit/utils': patch
+'@commercetools-uikit/fields': patch
+'@commercetools-uikit/inputs': patch
+'@commercetools-frontend/ui-kit': patch
+'@commercetools-uikit/design-system': patch
+'@commercetools-uikit/i18n': patch
```

Comment:
I'm getting different results when running changestets 🤔 perhaps it's worth double-checking, these are the packages in my output:
```
'@commercetools-uikit/localized-multiline-text-input': patch
'@commercetools-uikit/localized-rich-text-input': patch
'@commercetools-uikit/localized-text-input': patch
'@commercetools-uikit/search-text-input': patch
'@commercetools-uikit/rich-text-utils': patch
'@commercetools-uikit/select-utils': patch
'@commercetools-uikit/input-utils': patch
'@commercetools-uikit/money-input': patch
'@commercetools-uikit/time-input': patch
'@commercetools-uikit/calendar-utils': patch
'@commercetools-uikit/design-system': patch
```

Reply:
Hey @kark 

I re-run the changesets command and I get the same previous results.

Do you mean you only see those 11 packages in your changeset file?

Reply:
Hmm, that's right - this is the list (checking on 981296c76):
```
'@commercetools-uikit/localized-multiline-text-input': patch
'@commercetools-uikit/localized-rich-text-input': patch
'@commercetools-uikit/selectable-search-input': patch
'@commercetools-uikit/localized-text-input': patch
'@commercetools-uikit/search-text-input': patch
'@commercetools-uikit/rich-text-utils': patch
'@commercetools-uikit/select-utils': patch
'@commercetools-uikit/input-utils': patch
'@commercetools-uikit/money-input': patch
'@commercetools-uikit/radio-input': patch
'@commercetools-uikit/time-input': patch
'@commercetools-uikit/calendar-utils': patch
'@commercetools-uikit/design-system': patch
```

Reply:
So there might be something I'm missing.

For instance, `@commercetools-uikit/notifications` does appear in my list but not in yours. This package depends on `@commercetools-uikit/design-system` so I guess it should be in the list. But then I also have `@commercetools-uikit/i18n` in my list which does not depend on any local package, so it should not be in the list.

I've double checked my `changeset` installed version is `2.26.0`.

@emmenko Did anything like this happened to you before?

Reply:
Make sure to have the latest `main` pulled and that the branch is rebased.

<img width="431" alt="image" src="https://user-images.githubusercontent.com/1110551/223721606-273d056c-7f68-4f3a-b217-7089dd9257cd.png">